src/driver.jl

```diff
@@ -138,3 +138,41 @@ end
 
 copyfiles(drvname::AbstractString, new::AbstractString, old::AbstractString) =
     copyfiles(getdriver(drvname), new, old)
+
+"""
+    extensions()
+
+Returns a `Dict{String,String}` of all of the file extensions that can be read by GDAL, 
+with their respective drivers shortname.
+"""
+function extensions()
+    extdict = Dict{String,String}()
+    for i in 1:GDAL.gdalgetdrivercount()
+        driver = ArchGDAL.getdriver(i)
+        if !(driver.ptr == C_NULL)
+            exts = GDAL.gdalgetmetadataitem(driver.ptr, "DMD_EXTENSIONS", Cstring(C_NULL))
+            if !(exts isa Nothing)
+                for ext in split(exts)
```

Comment:
It's not immediately obvious what `split(exts)` does (since we don't specify a delimiter here). Can we add a clarifying comment for it?

Reply:
Yep. They come as a space delimited list from GDAL